frontend/pages/docs/users/managing-models-and-releases/upload-to-bailo/creating-a-model.mdx

```diff
@@ -0,0 +1,58 @@
+import Image from 'next/legacy/image'
+import Box from '@mui/material/Box'
+import bailoMarketIcon from 'public/docs/bailo_market_icon.png'
+import DocsWrapper from 'src/docs/DocsWrapper'
+
+## **Bailo Marketplace**
+
+The models added to Bailo are displayed in the Bailo Marketplace.
+
+<Box sx={{ maxWidth: '75%', margin: 'auto' }}>
+  <Image src={bailoMarketIcon} />
+</Box>
+
+This enables the models to be discoverable to other users. There is an exception to this, where the models are marked as
+private. In these instances the model will not be displayed.
+
+## **Finding models**
+
+Models are discoverable by the search functionality or via filtering on the tags.
+
+## **Adding a model to the Marketplace**
+
+New models can be added to Bailo using the the **_Add a new model_** button on the Marketplace, or alternatively, the
+**_Add Model_** button at the top of the application.
+
+Adding models to the Marketplace is quick and easy and requires three fields to be completed.
+
+The fields are:
+
+- **_*The Model Name and the Model Description.*_**
+
+These represent what is displayed for the model in the Marketplace. Public models are available for everyone, whereas
```

Comment:
"Public models are available for everyone, whereas
private models will only be visible if you have been given the right permissions in the model settings." should be moved below to replace "The choice of these will determine whether model is displayed in the Marketplace."